src/provider/provider_os_memory.c

```diff
@@ -0,0 +1,299 @@
+/*
+ * Copyright (C) 2022-2023 Intel Corporation
```

Comment:
Incorrect copyright date in `provider_os_memory.h`, `provider_os_memory.c` and `provider_os_memory.cpp` files. Or were those filed moved from some other repository?